discovery_engine_core/tooling/bin/clean_sources.rs

```diff
@@ -104,10 +120,10 @@ fn clean_sources(in_file: impl AsRef<Path>, out_file: impl AsRef<Path>) -> Resul
             bail!("missing domain field at {:?}", record.position());
         };
         if let Some(name) = record.get(0) {
-            if !name.is_empty() {
-                writer.write_byte_record(&[name, domain].as_ref().into())?;
-            } else {
+            if name.is_empty() {
```

Comment:
This is an arbitrary change that is out of scope.

Reply:
it's not arbitrary, but flagged by `clippy:pedantic`